src/spares/Apps/oc_capture.lsl

```diff
@@ -372,13 +378,22 @@ default
             }
         }
         
-        if (g_iCaptured && llGetAgentSize(g_kCaptor) == ZERO_VECTOR){
-            g_iReleaseTicks--;
-            if (g_iReleaseTicks == 599) llMessageLinked(LINK_SET,NOTIFY, "0You are out of range! Capture of secondlife:///app/agent/"+(string)g_kWearer+"/about will end in 10 minutes unless you come back!", g_kCaptor);
-            if (g_iReleaseTicks <= 0) UserCommand(CMD_OWNER,"capture",g_kCaptor);
-        } else if (g_iCaptured) {
-            if (g_iReleaseTicks < 600) llMessageLinked(LINK_SET,NOTIFY, "0Timer reset. Welcome Back!", g_kCaptor);
-            g_iReleaseTicks = 600;
+        if (g_iCaptured && g_iAutoRelease && llGetAgentSize(g_kCaptor) == ZERO_VECTOR){
```

Comment:
Is it really necessary to have a settings toggle to enable or disable the auto release?

Reply:
it was suggested in the discussion with roan. Why?

Reply:
I suggested it because there are some hardcore bondage people out there that might not want to be released automatically by anyone except the captor.  The standard grabber and many traps have this is an option, not automatic.